transport/model/project_member.go

```diff
@@ -0,0 +1,16 @@
+package model
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type ProjectMember struct {
```

Comment:
The response in the transport layer can be simplified to look like this. It returns all the information needed regarding the project member.